appveyor.yml

```diff
@@ -6,20 +6,20 @@ init:
 environment:
   matrix:
     - nodejs_version: '8'
-      webpack_version: latest
+      webpack_version: 3.8.1
       job_part: test
     - nodejs_version: '6'
-      webpack_version: latest
+      webpack_version: 3.8.1
       job_part: test
     - nodejs_version: '4.8'
-      webpack_version: latest
+      webpack_version: 3.8.1
```

Comment:
Latest is better here

Reply:
webpack v4 drops support for Node 4 and use es6, it will fail as long as we want to support Node v4.